lib/IRGen/GenDecl.cpp

```diff
@@ -29,7 +29,7 @@
 #include "swift/ClangImporter/ClangModule.h"
 #include "swift/Demangling/ManglingMacros.h"
 #include "swift/IRGen/Linking.h"
-#include "swift/Runtime/HeapObject.h"
+#include "../../stdlib/public/SwiftShims/RefCount.h"
```

Comment:
It was doing so already. The Runtime/HeapObject.h include there already transitively includes the stdlib/public/SwiftShims/RefCount.h header, by pulling in stdlib/public/SwiftShims/HeapObject.h. This may be wrong, but it's doing strictly less of that wrongness after this patch.